dmoj/cptbox/sandbox.py

```diff
@@ -103,7 +111,7 @@ def get_syscall_id(self, syscall):
     def readstr(self, address, max_size=4096):
         if self.address_bits == 32:
             address &= 0xFFFFFFFF
-        return super(AdvancedDebugger, self).readstr(address, max_size)
+        return super(AdvancedDebugger, self).readstr(address, max_size).decode('utf-8')
```

Comment:
Bad design. Return bytes.

Reply:
We still need to either convert fs regex to bytes, or this to bytes

Reply:
This is good enough for now. We should catch UnicodeDecodeError and kill the subprocess, since potential security hole.